src/components/VisualizationPlugin/VisualizationPluginWrapper.js

```diff
@@ -27,10 +28,39 @@ const VisualizationPluginWrapper = (props) => {
         [pluginProps]
     )
 
+    useEffect(() => {
+        setPluginProps({
+            displayProperty: props.displayProperty,
+            visualization: props.visualization,
+            filters: props.filters,
+            forDashboard: props.forDashboard,
+            id: props.id,
+            isInModal: props.isInModal,
+            style: props.style,
+            onChartGenerated: props.onChartGenerated,
```

Comment:
I don't think all these callbacks are needed or used by dashboard/interpretations modal.
Perhaps we should keep this to the minimum required to avoid confusion?